ecosystem/sep-0006.md

```diff
@@ -410,7 +410,8 @@ Request Parameters:
 | `memo`                        | string                  | (optional) Value of memo to attach to transaction, for `hash` this should be base64-encoded. Because a memo can be specified in the SEP-10 JWT for [Shared Accounts](#shared-omnibus-or-pooled-accounts), this field as well as `memo_type` can be different than the values included in the SEP-10 JWT. For example, a client application could use the value passed for this parameter as a reference number used to match payments made to `account`.              |
 | `email_address`               | string                  | (optional) Email address of depositor. If desired, an anchor can use this to send email updates to the user about the deposit.                                                                                                                                                                                                                                                                                                                                        |
-| `type`                        | string                  | (optional) Type of deposit. If the anchor supports multiple deposit methods (e.g. `SEPA` or `SWIFT`), the wallet should specify `type`. This field may be necessary for the anchor to determine which KYC fields to collect.                                                                                                                                                                                                                                          |
+| `funding_method`              | string                  | (optional) Method supported by the anchor for transferring or settling assets. If the anchor supports multiple deposit methods (e.g. `SEPA` or `SWIFT`), the wallet should specify `funding_method`. This field may be necessary for the anchor to determine which KYC fields to collect.                                                                                                                                                                             |
+| `type`                        | string                  | (**deprecated** in favor of `funding_method` ,optional) Type of deposit. If the anchor supports multiple deposit methods (e.g. `SEPA` or `SWIFT`), the wallet should specify `type`. This field may be necessary for the anchor to determine which KYC fields to collect.                                                                                                                                                                                             |
```

Comment:
Something we could do to make things more readable is to move all deprecated fields to the bottom on the list?

Eventually we do want to remove these deprecated fields but I want to be intentional about when that happens and why its the right time.